recipes/ecell4_base/cmake-generator-vs2017.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index 48d35db8..c6e3a2ed 100644
+--- a/setup.py
++++ b/setup.py
+@@ -46,6 +46,8 @@ class CMakeBuild(build_ext):
+ 
+         if platform.system() == "Windows":
+             cmake_args += ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY_{}={}'.format(cfg.upper(), extdir)]
++            # force cmake to use VS15 2017 with no respect for environment variables
++            cmake_args += ['-G', 'Visual Studio 15 2017']
```

Comment:
This builds a  32-bit library.

Reply:
I don't know much about CMake and MSVC, but doesn't `-A x64` make target architecture 64-bit? The option is given [in the setup.py](https://github.com/ecell/staged-recipes/blob/ecell4_base/recipes/ecell4_base/cmake-generator-vs2017.patch#L12). I referenced this (https://cmake.org/cmake/help/v3.15/generator/Visual%20Studio%2015%202017.html). Do you know the correct way to specify the architecture?